Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -177,6 +177,7 @@ def addSeeding(
     seedFilterConfigArg: SeedFilterConfigArg = SeedFilterConfigArg(),
     spacePointGridConfigArg: SpacePointGridConfigArg = SpacePointGridConfigArg(),
     seedingAlgorithmConfigArg: SeedingAlgorithmConfigArg = SeedingAlgorithmConfigArg(),
+    houghTransformConfig: acts.examples.HoughTransformSeeder.Config = acts.examples.HoughTransformSeeder.Config(),
```

Comment:
this is a bit inconsistent with the other config args because it is not wrapped in a named tuple. just wanted to point it out dont think it is necessarily blocking

Reply:
I know. Though, it did not seem to me relevant given the fact that we are going to experiment with it (including config changes) a lot still.